src/frontend-admin/package-lock.json

```diff
@@ -2776,7 +2776,8 @@
             },
             "ansi-regex": {
               "version": "2.1.1",
-              "bundled": true
+              "bundled": true,
+              "optional": true
```

Comment:
I'm curious, what version of Node are you running? Generally changes in package-lock.json indicate differing node versions among devs (I might be the one who committed with an outdated Node :) )

Reply:
I'm running v12.2.0

Reply:
Ah gotcha - latest and greatest. Not a big deal at all but I usually recommend sticking with the LTS version of Node, which I think is 10.16 right now. No need to change anything on your end though - I wager you will not be `npm i`-ing very often in this project.